src/components/molecules/Visualizer/Plugin/types.ts

```diff
@@ -269,6 +269,14 @@ export type UI = {
     extended?: boolean | undefined,
   ) => void;
   readonly close: () => void;
+  readonly setPosition: (options: SetUIPositionOptions) => void;
+};
+
+export type SetUIPositionOptions = {
+  zone: "outer" | "inner";
+  section: "left" | "center" | "right";
+  area: "top" | "middle" | "bottom";
+  method?: "insert" | "append";
```

Comment:
Do you think we should use `method` here or some other naming?

Reply:
good